rpm/rpm.go

```diff
@@ -61,10 +61,15 @@ var archToRPM = map[string]string{
 }
 
 func ensureValidArch(info *nfpm.Info) *nfpm.Info {
-	arch, ok := archToRPM[info.Arch]
-	if ok {
-		info.Arch = arch
+	if info.APK.APKArch != "" {
+		info.Arch = info.APK.APKArch
```

Comment:
Guessing this should be `info.RPM.Arch`?

Reply:
Correct